libkbfs/root_metadata.go

```diff
@@ -1089,6 +1089,9 @@ func (rmds *RootMetadataSigned) MakeFinalCopy(
 	}
 	// Set the final flag.
 	newBareMd.SetFinalBit()
+	// Set the copied bit, so that clients don't take the ops and byte
+	// counts in it seriously.
+	newBareMd.SetWriterMetadataCopiedBit()
```

Comment:
I guess it's not really a regression test, but there's a new kbfs-docker test that will catch this (though those parts are currently commented out since they don't work).  That's how I noticed this issue in the first place actually.

If you'd still like a test in libkbfs, I can do that separately later, but I'm going to merge as is for now so I can vendor into server.